File: publisher.js
const Ably = require('ably');
const constants = require('./constants');

/* Instance the Ably library */
const realtime = new Ably.Realtime({key: constants.API_KEY});
const channelOpts = {cipher: {key: constants.SECRET}};
const pubChannel = realtime.channels.get('encrypted:messages', channelOpts);

console.log('Publishing encrypted message');

/* Publish encrypted messages */
pubChannel.publish('unencrypted', 'an encrypted message');

File: src/main/java/io/ably/tutorial/Publisher.java
package io.ably.tutorial;

import static io.ably.tutorial.Constants.API_KEY;

import io.ably.lib.realtime.AblyRealtime;
import io.ably.lib.types.AblyException;

public class Publisher {
    
    public static void main(String[] args) throws AblyException {
        AblyRealtime ablyRealtime = new AblyRealtime(API_KEY);
    }
    
}
